packages/svelte/src/internal/client/runtime.js

```diff
@@ -155,6 +158,10 @@ export function set_dev_current_component_function(fn) {
 	dev_current_component_function = fn;
 }
 
+export function increment_version() {
+	return current_version++;
```

Comment:
I'm a bit worried that this will crash your app if it runs long enough. How often will this run, is it realistic that an integer overflow happens eventually?

Reply:
It would take several years of hundreds of updates a second before we even reached `Number.MAX_SAFE_INTEGER`